packages/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -187,7 +183,7 @@ function createCompiler(webpack, config, appName, urls, useYarn) {
       );
       console.log(
         'To ignore, add ' +
-          chalk.cyan('// tslint:disable-next-line') +
+          chalk.cyan('// eslint-disable-next-line') +
```

Comment:
This should remain `tslint:disable-next-line` - we're not using ESLint.

Reply:
I think this is an unneeded change isn't it, as we still use the published `react-dev-utils` not a ts specific one?

Reply:
Oh - yeah, my bad,